modeling-cmds/src/ok_response.rs

```diff
@@ -75,6 +75,13 @@ define_ok_modeling_cmd_response_enum! {
             pub entity_ids: Vec<Uuid>,
         }
 
+        /// The response from the `Loft` command.
+        #[derive(Debug, Serialize, Deserialize, JsonSchema, ModelingCmdOutput)]
+        pub struct Loft {
+            ///The UUID of the newly created solid loft.
```

Comment:
This is a solid3d right?

Reply:
yes, but the reader should be able to infer that a solid loft is a solid 3d.  there's no such thing as a solid lofted 2D in our engine